src/dashboards/flow_deep_dive.json

```diff
@@ -323,11 +406,19 @@
         }
       ],
       "title": "Data Usage by Application",
-      "transform": "table",
+      "transformations": [
```

Comment:
The `Merge` transformations at the summary tables must be removed. There seems a bug that prevents the value in the DSCP column to display properly.

Was there a specific reason for adding the `Merge` transformations?
![Screenshot from 2021-05-27 21-16-16](https://user-images.githubusercontent.com/540079/119886000-ce93e100-bf32-11eb-96f2-59fd5f2fd454.png)
![Screenshot from 2021-05-27 21-16-25](https://user-images.githubusercontent.com/540079/119886004-cf2c7780-bf32-11eb-83bf-1532802c28b9.png)

Reply:
Not a specific reason for Merge. It was added when the dashboards updated to a more recent Grafana.  As Helm 7 now relies on Grafana 7.5, I left in dashboard schema changes generated by Grafana 7.5.